src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$headerPadding: 30px;
```

Comment:
think you should follow naming like that:

$header-padding;
$title-size-ipad;
$title-size-desktop;
etc.....

it's easier to read compare to camelcase naming :)